src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/DefendantResponseCaseHandedOfflineRespondentNotificationHandler.java

```diff
@@ -40,32 +49,81 @@
 
     @Override
     public String camundaActivityId(CallbackParams callbackParams) {
-        return TASK_ID;
+        return isRespondent1(callbackParams) ? TASK_ID_RESPONDENT1 : TASK_ID_RESPONDENT2;
     }
 
     @Override
     public List<CaseEvent> handledEvents() {
         return EVENTS;
     }
 
+    //Offline notification will point to a new MP template for displaying defendant responses
     private CallbackResponse notifyRespondentSolicitorForCaseHandedOffline(CallbackParams callbackParams) {
         CaseData caseData = callbackParams.getCaseData();
+        String recipient;
+        String templateID;
 
+        //Use 1v1 Template
+        if (is1v1Or2v1Case(caseData)) {
+            recipient = caseData.getRespondentSolicitor1EmailAddress();
+            templateID = notificationsProperties.getSolicitorDefendantResponseCaseTakenOffline();
+        } else {
+            //Use Multiparty Template as there are 2 defendant responses
+            templateID = notificationsProperties.getSolicitorDefendantResponseCaseTakenOfflineMultiparty();
+            if (isRespondent1(callbackParams)) {
+                recipient = caseData.getRespondentSolicitor1EmailAddress();
+            } else {
+                recipient = caseData.getRespondentSolicitor2EmailAddress();
+            }
+        }
+
+        sendNotificationToSolicitor(caseData, recipient, templateID);
+
+        return AboutToStartOrSubmitCallbackResponse.builder().build();
+    }
+
+    private void sendNotificationToSolicitor(CaseData caseData, String recipient, String templateID) {
         notificationService.sendMail(
-            caseData.getRespondentSolicitor1EmailAddress(),
-            notificationsProperties.getSolicitorDefendantResponseCaseTakenOffline(),
+            recipient,
+            templateID,
             addProperties(caseData),
             String.format(REFERENCE_TEMPLATE, caseData.getLegacyCaseReference())
         );
+    }
 
-        return AboutToStartOrSubmitCallbackResponse.builder().build();
+    private Boolean isRespondent1(CallbackParams callbackParams) {
+        CaseEvent caseEvent = CaseEvent.valueOf(callbackParams.getRequest().getEventId());
+        return caseEvent.equals(NOTIFY_RESPONDENT_SOLICITOR1_FOR_CASE_HANDED_OFFLINE);
+    }
+
+    private boolean is1v1Or2v1Case(CaseData caseData) {
+        return getMultiPartyScenario(caseData).equals(ONE_V_ONE) || getMultiPartyScenario(caseData).equals(TWO_V_ONE);
     }
 
     @Override
     public Map<String, String> addProperties(CaseData caseData) {
-        return Map.of(
-            CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference(),
-            REASON, caseData.getRespondent1ClaimResponseType().getDisplayedValue()
-        );
+        if (getMultiPartyScenario(caseData).equals(ONE_V_ONE)) {
```

Comment:
if we move this logic to PartyUtils.java in a static method, we can easily test it in PartyUtilsTest, rather than big tests added for various scenarios to DefendantResponseCaseHandedOfflineRespondentNotificationHandlerTest.java.
This way is adopted in some of the notification handlers and we are working on same pattern in other notification tickets e.g CMC-1864 (see https://github.com/hmcts/civil-service/pull/655)